libimage/filters.go

```diff
@@ -152,6 +152,13 @@ func (r *Runtime) compileImageFilters(ctx context.Context, options *ListImagesOp
 			}
 			filter = filterReadOnly(readOnly)
 
+		case "manifest", "image":
```

Comment:
Filters have the form `key=value`, so `--reference manifest` would error out above when parasing.  Can we instead have `manifests={true,false}`?